src/features/virtualized-table/mapRowToCellData.js

```diff
@@ -0,0 +1,17 @@
+// @flow
```

Comment:
The purpose of this helper is to make sure the cell is getting the correct data to render whatever it needs. However, virtualizedTable provides a prop `rowGetter` that allows you to format that data to your desire.

@jpfranco and I had a discussion on this and we both agree that we will use the rowGetter prop to update the data moving forward. Let me know if you need help on migrating to use `rowGetter`

Reply:
I also believe this is not used anywhere.